docs/userman/advanced_features/auth/authentication.md

```diff
@@ -427,3 +427,73 @@ connect to the directory service:
         # Returns a valid Credentials instance if the LDAP server validates the password
         # "pass123" for user "user1". Raises an InvalidCredentials exception otherwise.
         ```
+
+# Microsoft Entra ID Authenticator
+
+Taipy also provides support for Microsoft Entra ID authentication.
+
+An authenticator using the *Entra ID* protocol has two specific parameters that need to be provided
+in order to properly connect to the Microsoft Entra ID service:
+
+- *client_id*: The client ID of the Entra ID application. The application must be registered in the
+    Azure Entra ID portal and have the required permissions including the "User.Read" and
+    "GroupMember.Read.All" permissions.
+- *tenant_id*: The tenant ID of the Entra ID organization.
```

Comment:
I would recommend we indicate where and how to read or configure these two Entra ID settings.

Reply:
I have no idea where to find this info. Do you know?